src/main/java/com/labsynch/labseer/service/ChemStructureService.java

```diff
@@ -26,12 +26,10 @@
 	public int[] searchMolStructures(String molfile, String structureTable,
 			String plainTable, String searchType, Float simlarityPercent) throws CmpdRegMolFormatException;
 
-	public boolean dropJChemTable(String tableName);
-
 	public int[] searchMolStructures(String molfile, String structureTable,
 			String plainTable, String searchType) throws CmpdRegMolFormatException;
 
-	public boolean createJChemTable(String tableName, boolean tautomerDupe);
+	public boolean truncateStructureTable(String structureTable);
```

Comment:
That's great to remove all these mentions of JChem and combine them here. I was worried these functions were used in the flyway migrations that create JChem tables but those seem to have their own functions defined locally: https://github.com/mcneilco/acas-roo-server/blob/master/src/main/java/com/labsynch/labseer/db/migration/postgres/jchem/V2_0_0_7__Create_Jchem_Tables.java